modules/images/regenerate-existing-images/helper.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Functions used by the background process API.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Creates a background job.
+ *
+ * Job refers to the term and queue refers to the `background_job` taxonomy.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $name Name of job identifier.
+ * @param array  $data Data for the job.
```

Comment:
See my other comment, what can be put into `$data`? Are there any predefined keys we expect, or is it entirely arbitrary? If there are certain keys that are supported, we should document those here.